docs/conf.py

```diff
@@ -372,21 +359,23 @@ def __add__(self, other):
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
 
+def RunSphinxAPIDoc(_):
+  """Runs sphinx-apidoc to auto-generate documentation."""
+  current_directory = os.path.abspath(os.path.dirname(__file__))
+  module = os.path.join(current_directory, '..', 'plaso')
```

Comment:
module => module_path or module_directory

also maybe use os.path.dirname() instead of '..'

Reply:
Done